templates/core/features/secure-boot.html

```diff
@@ -0,0 +1,150 @@
+{% extends "core/base_core.html" %}
+
+{% block title %}Secure boot | Ubuntu Core{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1ltH7T2boIJ7VcUni4Kmc0XGlmmxWAf5u82LEwkEGP24/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row u-equal-height">
+    <div class="col-7">
+      <h1>Secure boot</h1>
+      <p class="p-heading--4">Protect against vulnerabilities at boot time</p>
+      <p>Computers are vulnerable during the boot process if they are not secured. The kernel, hardware peripherals and user space processes are all initiated at boot and any vulnerability in the boot firmware can have cascading effects on the entire system.</p>
+      <p>In the case of an attack on boot firmware, damages are so profound that often hardware replacement is the only fix. In an industrial IoT scenario, this means considerable downtime, manual maintenance, possibly at several locations and CapEx for hardware replacement. An utterly undesirable outcome.</p>
+      <p>
+        <a href="/core#get-in-touch" class="p-button--positive js-invoke-modal">Improve security at boot time with Ubuntu CoreSecure your devices</a>
```

Comment:
```suggestion
        <a href="/core/docs/security-and-sandboxing">Learn about security and sandboxing in Ubuntu Core&nbsp;&rsaquo;</a>
```

I made Galem update this CTA.